atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraServer.java

```diff
@@ -60,8 +64,4 @@ static CassandraServer of(String hostName, InetSocketAddress addr) {
     static CassandraServer of(String hostName, Set<InetSocketAddress> reachableProxies) {
         return ImmutableCassandraServer.of(hostName, reachableProxies);
```

Comment:
Making a note for myself, may want test and use [Caffeine Interner](https://github.com/ben-manes/caffeine/wiki/Interner) instead of the Guava Interner